includes/blocks/class-sensei-block-take-course.php

```diff
@@ -139,7 +139,7 @@ private function render_disabled_with_prerequisite( $course_id, $content ) {
 	 *
 	 * @return string
 	 */
-	private function get_course_prerequisite_message( $course_id ) {
+	public function get_course_prerequisite_message( $course_id ) {
```

Comment:
Why do we need to turn this to public?

Reply:
It's used by the WCPC Purchase button which displays this message a bit differently right now.